### PR TITLE
[Hub Generated] Review request for Microsoft.IoTCentral to add version stable/2018-09-01

### PR DESCRIPTION
**Pull request by kawayiboy**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/kawayiboy/azure-rest-api-specs/telu%2F400errs...master/).

### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/iotcentral/resource-manager/Microsoft.IoTCentral/stable/2018-09-01/examples/Apps_CreateOrUpdate.json [hunk 1]
```diff
@@ -67,6 +67,12 @@
       }
     },
     "202": {},
+    "400": {
```
Comment (anthony-c-martin) [inline specification/iotcentral/resource-manager/Microsoft.IoTCentral/stable/2018-09-01/examples/Apps_CreateOrUpdate.json]: This change should not be necessary - 400 is already covered by the `default` case below, which uses the same error contract. What's the motivation for the change?

**Review by anthony-c-martin — CHANGES_REQUESTED** [review]

**Comment by NullMDR** [comment]
@kawayiboy Is there any update on this PR?

**Comment by ravbhatnagar** [comment]
no response from PR owner on the earlier ARM review comment. removing ARM review label.

**Comment by NullMDR** [comment]
@kawayiboy Any update? please resolve merge conflict first

**Comment by NullMDR** [comment]
@kawayiboy Any update on this PR?

**Comment by NullMDR** [comment]
@kawayiboy Any update on this PR?
